elastic4s-core/src/main/scala/com/sksamuel/elastic4s/requests/indexes/IndexTemplateHandlers.scala

```diff
@@ -82,7 +82,7 @@ object CreateIndexTemplateBodyFn {
       create.settings.foreach {
         case (key, value) => builder.autofield(key, value)
       }
-      builder.rawField("analysis", AnalysisBuilder.build(create.analysis.get))
+      if(create.analysis.nonEmpty) builder.rawField("analysis", AnalysisBuilder.build(create.analysis.get))
```

Comment:
how about avoiding the whole `.get` with:
```
      create.analysis.foreach { analysis =>
          builder.rawField("analysis", AnalysisBuilder.build(analysis))
      }
```

Reply:
Yep that would be better.

Reply:
`.foreach` sounds good. Let me change it.

Reply:
I've updated it.